packages/react/src/common/useIsAfterBreakpoint.tsx

```diff
@@ -3,37 +3,39 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { useLayoutEffect, useState } from 'react'
+import { useEffect, useState } from 'react'
 
 // TODO: we should set the breakpoint in JS somewhere and render this and the sass variables from that
 export const BREAKPOINTS = {
   medium: '37.5rem',
   wide: '72.5rem',
-}
+} as const
 
-type useIsAfterBreakpointProps = 'medium' | 'wide'
+type Breakpoint = keyof typeof BREAKPOINTS
 
-const useIsAfterBreakpoint = (breakpoint: useIsAfterBreakpointProps) => {
+const useIsAfterBreakpoint = (breakpoint: Breakpoint): boolean => {
```

Comment:
The return type is inferred here, I don't think we need to define it. 

```suggestion
const useIsAfterBreakpoint = (breakpoint: Breakpoint) => {
```